src/main/java/rx/Completable.java

```diff
@@ -1971,19 +1972,40 @@ public final void subscribe(CompletableSubscriber s) {
         } catch (NullPointerException ex) {
             throw ex;
         } catch (Throwable ex) {
+            Exceptions.throwIfFatal(ex);
             ERROR_HANDLER.handleError(ex);
```

Comment:
Ah, ok. I also see that it's supposed to not call the hook, but it does. However, couldn't be the case that multiple calls to the error handler are made for the same error (e.g., due to operators)?